projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/Utils.kt

```diff
@@ -181,7 +180,7 @@ const val RANDOM_PASSWORD_LEN = 11
 
 fun generatePassword(): String {
   val charPool = ('a'..'z') + ('A'..'Z') + ('0'..'9')
-  return generateSequence { charPool.random() }
+  return generateSequence { charPool.random(SecureRNG()) }
```

Comment:
Oh, I remember I bumped into the same problem some time ago, but it turned out that there was `.asKotlinRandom()` method that solved the task quite concise. So no need to create our own wrapper.

Also, I advise moving construction out of the lambda to avoid creating it multiple times.

So it could look like this as the result:

```suggestion
  val random = SecureRandom().asKotlinRandom()
  return generateSequence { charPool.random(random) }
```

Reply:
Oops, I've missed this useful function, thanks! Of course your solution is much better.